cmake/scripts/common/AddonHelpers.cmake

```diff
@@ -199,6 +198,8 @@ macro (build_addon target prefix libs)
       endif()
     endif()
 
+    string(REPLACE "<platform>\@PLATFORM\@</platform>" "<platform>@PLATFORM_TAG@</platform>" addon_file "${addon_file}")
```

Comment:
Should I keep this or are addon maintainers forced to adjust their `addon.xml.in` before it can be accepted into the binary addons repo?

Reply:
Can we do some grace period that we support both between now and just before final so they can adjust over coming month?

Reply:
Yes, but we can't let the ones using the old style into the binary repo without this hack.
e.g.: old style is android, which would match `android-armv7` and `android-aarch64`

Reply:
It's more for transition period between shipping it with core and moving it to repo

Reply:
Bundled binary add-ons will always work (no hack & not updated **OR** no hack & update **OR** with the hack).

Reply:
I'd say keep it for v18, but remove for v19